bootstrap/sql/migrations/native/1.5.0/mysql/schemaChanges.sql

```diff
@@ -148,3 +148,12 @@ JSON_EXTRACT(json, '$.sourceConfig.config.dbtConfigSource.dbtSecurityConfig.gcpC
 JSON_EXTRACT(json, '$.sourceConfig.config.dbtConfigSource.dbtSecurityConfig.gcpConfig.externalType') OR 
 JSON_EXTRACT(json, '$.sourceConfig.config.dbtConfigSource.dbtSecurityConfig.gcpConfig.path')
 ) is NULL AND JSON_EXTRACT(json, '$.sourceConfig.config.dbtConfigSource.dbtSecurityConfig.gcpConfig') is not null;
+
+ALTER TABLE glossary_term_entity ADD COLUMN fullyQualifiedName VARCHAR(768) GENERATED ALWAYS AS (json ->> '$.fullyQualifiedName') NOT NULL;
```

Comment:
adding a column like this here, will defeat the fqnHash changes. We end up limiting the max size of a glossary term because varchar is set to 768 cc @mohityadav766

Reply:
yes @harshach will not be having any column for fullyQualifiedName